docs/overview/installation.md

```diff
@@ -16,43 +19,13 @@
 3. `BASE_DIR` setting is required. Ensure that it is defined:
 
     ```python
+    # app/settings.py
     from pathlib import Path
 
     BASE_DIR = Path(__file__).resolve().parent.parent
     ```
 
-4. _Optional._ Set [`COMPONENTS.dirs`](../reference/settings.md#django_components.app_settings.ComponentsSettings.dirs)
```

Comment:
Since this installation step is optional, I've moved it down [here](https://github.com/django-components/django-components/pull/1166/files#diff-54caa5c9093caf057439e1e8d510a8d2131298fb28516a36a551e220a7bfb636R139) into the "Optional" section.